src/main/java/net/neoforged/neoforge/event/entity/player/CriticalHitEvent.java

```diff
@@ -74,19 +73,36 @@ public boolean isCriticalHit() {
     }
 
     /**
-     * Changes the critical hit state.
-     * 
+     * Changes the critical hit state. If set critical hit to true, disables sweep attack
```

Comment:
~~If set critical hit to true, disables sweep attack~~ -> If set to true, `disableSweep` will also be set to true.

Reply:
fixed